lib/fastlane/actions/xcodebuild.rb

```diff
@@ -95,7 +99,7 @@ def self.run(params)
             end
 
             # Store IPA path for later deploy steps (i.e. Crashlytics)
-            Actions.lane_context[SharedValues::IPA_OUTPUT_PATH] = params[:export_path] + "." + params[:export_format].downcase
```

Comment:
During review I've noticed that removing default value for `:export_format` could break Mac builds. Please wait for the fix.